app/include/user_config.h

```diff
@@ -11,11 +11,13 @@
 // #define FLASH_16M
 #define FLASH_AUTOSIZE
 #define FLASH_SAFE_API
-// #define DEVELOP_VERSION
-#define FULL_VERSION_FOR_USER
 
-#define USE_OPTIMIZE_PRINTF
+// Byte 107 of esp_init_data_default, only one of these 3 can be picked
+#define ESP_INIT_DATA_ENABLE_READVDD33
+//#define ESP_INIT_DATA_ENABLE_READADC
```

Comment:
My experience with trying to build was extremely memorable, with an unsuccessful outcome. I spent a lot of time  , and that's because I'm no kind of expert. I finally gave up, there were too many gotcha's with setting up Docker, VM, gitHub etc. Lots of hidden stuff an expert would catch, but a novice doesn't even know to look for. Its best to avoid that route.In my opinion, its far more preferable to stay with a frightanic build and then use the esp tool to change the so-called byte 107 to the way you want it.Based on expert suggestions I wrote down what I did in #1007. It enables me to stay consistent with the SDK and Dev/Master in one fell swoop as you don't have to load the SDK 1.4  separately. If you already have the right SDK, the notes in #911 walk you through the byte 107 change.
 

      From: Jarrod <notifications@github.com>
 To: nodemcu/nodemcu-firmware <nodemcu-firmware@noreply.github.com> 
Cc: fixingthingsguy <sbijanki@Yahoo.com>
 Sent: Friday, February 5, 2016 11:18 PM
 Subject: Re: [nodemcu-firmware] Remove adc.readvdd33() limitation. (#746)
   
In app/include/user_config.h:>  
> -#define USE_OPTIMIZE_PRINTF
> +// Byte 107 of esp_init_data_default, only one of these 3 can be picked
> +#define ESP_INIT_DATA_ENABLE_READVDD33
> +//#define ESP_INIT_DATA_ENABLE_READADC
I would like to build the firmware with this option , what or how is the best root to take .—
Reply to this email directly or view it on GitHub.